pom.xml

```diff
@@ -64,6 +64,11 @@
 			<artifactId>mapstruct</artifactId>
 			<version>${mapstruct.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate</groupId>
```

Comment:
You could also add `spring-boot-starter-validation` instead.

Reply:
done